routes.js

```diff
@@ -0,0 +1,12 @@
+async function routes(fastify, options) {
+	fastify.get("/health", async (request, reply) => {
+		return reply.code(200).send({ status: "ok" });
+	});
+
+	fastify.get("/health/db", async (request, reply) => {
```

Comment:
```suggestion
    fastify.get('/health', async (_request, reply) => {
```
if you don't use the first variable but have to write it to access the second param, add `underscore` it tells it you know it's not used but have to be there